### DISCUSSION TITLE
A little coding help please - Audio Files

### DISCUSSION THREAD

**Discussion by bjazmoore** [hajimehoshi/ebiten#1519] [Q&A] [answered]
I have been studying the code for the Audio example (https://ebiten.org/examples/audio.html).  I think I kind of understand some of it, but there is a lot going on.  I was hoping maybe you could whip up the absolute minimum code to play a WAV file called music.wav in the same folder as main.go using these two libraries:

         "github.com/hajimehoshi/ebiten/v2/audio"
	"github.com/hajimehoshi/ebiten/v2/audio/wav"

I am not sure the first one would be required.  I just want the go code to open the wav file and play it and then quit.

Huge thanks in advance.

**Comment by hajimehoshi** [comment, marked as answer]
https://github.com/hajimehoshi/ebiten/blob/master/examples/wav/main.go is the better example to perform playing a wav file with a minimal code.

**Reply by bjazmoore** [reply]
I was able to embbed my wav file and use a reader to read it as a byte slice.  I changed the code to start playing when the program is launched, continuously loop and start and stop when "P" is pressed.  Thanks for the great example code.  Not sure how I missed it.

**Comment by bjazmoore** [comment]
Am I able to mark these discussions as answered?  Trying to figure out how.  Thanks everyone!

**Reply by bjazmoore** [reply]
Never mind - I just figured it out!
